ragaai_catalyst/tracers/agentic_tracing/upload/trace_uploader.py

```
@@ -49,11 +50,13 @@
     from ragaai_catalyst.tracers.agentic_tracing.upload.upload_code import upload_code
     # from ragaai_catalyst.tracers.agentic_tracing.upload.upload_trace_metric import upload_trace_metric
     from ragaai_catalyst.tracers.agentic_tracing.utils.create_dataset_schema import create_dataset_schema_with_trace
+    from ragaai_catalyst.tracers.agentic_tracing.upload.session_manager import session_manager
     from ragaai_catalyst import RagaAICatalyst
```

Ruff (0.11.9)

`ragaai_catalyst.RagaAICatalyst` imported but unused; consider using `importlib.util.find_spec` to test for availability

(F401)